dts/bindings/clock/fixed-clock.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2019 Derek Hageman <hageman@inthat.cloud>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: Generic fixed rate clock provider
+version: 0.1
+
+description: >
+    This is a representation of a generic fixed rate clock provider.
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "fixed-clock"
+
```

Comment:
Don't we miss a "clocks-controller" property here?

Reply:
It appears that `clocks-controller` is not spec'd in any upstream binding.  Do we need it in Zephyr?  Guessing we use it in the generator.

Reply:
Hmm, doesn't look like we use it in the generator from a simple grep.

Reply:
No, I agree, we are not using it. We use `fixed-clock` and then `clock-frequency`